.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+## How to Contribute?
+
+> To get started...
+
+1.  🍴 [Fork this repo](https://github.com/CircuitVerse/Interactive-Book)
+2.  🔨 Hack away
+3.  👥 Add yourself as a contributor under the credits section
```

Comment:
I think we must remove this line as the images of contributions are automatically fetched and not hard-coded. This can be confusing i guess?
What do you think?

Reply:
Okay If that looks confusing, I will remove it. Thanks for the review.